doc/DeveloperGuide.md

```diff
@@ -94,6 +94,8 @@ Use case ends.
 2. Should be able to hold up to 1000 persons.
 3. Should come with automated unit tests and open source code.
 4. Should favor DOS style commands over Unix-style commands.
+5. Should require none or minimal mouse control.
```

Comment:
Not sure whether this is non-functional?

Reply:
This seems functional. Being CLI-based is a core functionality of the AddressBook.